packages/extension-ui/src/components/Button.tsx

```diff
@@ -44,54 +44,26 @@ function Button ({ children, className, isDisabled, label, onClick, to }: Props)
 
 export default styled(Button)`
   box-sizing: border-box;
-  display: ${({ isSmall }) =>
-    isSmall
-      ? 'inline-block'
-      : 'block'
-  };
+  display: ${({ isSmall }): string => isSmall ? 'inline-block' : 'block'};
   margin: ${defaults.boxMargin};
   padding: ${defaults.boxPadding};
-  width: ${({ isSmall }) =>
-    isSmall
-      ? 'auto'
-      : '100%'
-  };
+  width: ${({ isSmall }): string => isSmall ? 'auto' : '100%'};
 
   button {
-    background: ${({ isDanger }) =>
-      isDanger
-        ? defaults.btnBgDanger
-        : defaults.btnBg
-    };
-    border: ${defaults.btnBorder}${({ isDanger }) =>
-      isDanger
-        ? defaults.btnColorDanger
-        : defaults.btnColor
-    };
+    background: ${({ isDanger }): string => isDanger ? defaults.btnBgDanger : defaults.btnBg};
```

Comment:
Ok, this really sucks.

The issue is that eslint complains about the indent, so actually would like this (when multi-line) -

```js
background: ${({ isDanger }) =>
  isDanger
    ? defaults.btnBgDanger
    : defaults.btnBg
```

This is due to the fact that this is nested inside `button { ... }`